cmd/kubeadm/app/preflight/checks.go

```diff
@@ -607,3 +607,20 @@ func TryStartKubelet() {
 		}
 	}
 }
+
+// RunPreFlightChecks runs all pre-flight checks
+func RunPreFlightChecks(cfg *kubeadmapi.MasterConfiguration) error {
+	fmt.Println("[preflight] Running pre-flight checks")
+
+	// First, check if we're root separately from the other preflight checks and fail fast
+	if err := RunRootCheckOnly(); err != nil {
```

Comment:
Agreed. That would then eliminate the need for the new function I've created (`RunPreFlightChecks`). Just wanted to clarify the intention.